app/validators/url_validator.rb

```diff
@@ -0,0 +1,16 @@
+class UrlValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return unless value.present?
```

Comment:
could we combine `return unless value.present?` and `..unless url_valid?(value)` in single condition?

Reply:
Sure, I will make two separate guard clauses. I think 2 will be more readable.